### ISSUE TITLE
Annotation in google geo chart

### ISSUE THREAD

**Issue by bhar24** [google/google-visualization-issues#2076]
I want to show the annotation to google geo chart similar to bar, line charts etc. But currently i can add/modify tooltip only. Does google geo chart support annotation? If not, is there any other option to show data value to regions?

**Comment by grabks** [comment]
No, GeoCharts currently do not support annotations. You might be able to
use 'text' mode to get something similar to what you want.
